requestor-tutorials/service-development/service-example-6-external-api-request.md

```diff
@@ -0,0 +1,183 @@
+---
+description: >-
+  Example showing how to make REST call to an external public API from VM running on a Provider node.
+---
+
+# Service Example 6: External API Request
+
+{% hint style="info" %}
+The example depicts the following features:
+
+* [Outbound Network](../vm-runtime/computation-payload-manifest.md#compmanifestnetinetout--object)
+* [Computation Payload Manifest](../vm-runtime/computation-payload-manifest.md)
+{% endhint %}
+
+{% hint style="info" %}
+Full code of the example is available in the yapapi repository: [https://github.com/golemfactory/yapapi/tree/master/examples/external-api-request](https://github.com/golemfactory/yapapi/tree/master/examples/external-api-request)
+{% endhint %}
+
+## Prerequisites
+
+As with the other examples, we're assuming here you already have your [yagna daemon set-up to request the test tasks](../flash-tutorial-of-requestor-development/) and that you were able to [configure your Python environment](../flash-tutorial-of-requestor-development/run-first-task-on-golem.md) to run the examples using the latest version of `yapapi`. If this is your first time using Golem and yapapi, please first refer to the resources linked above.
+
+This example involves [Computation Payload Manifest](../../requestor-tutorials/vm-runtime/computation-payload-manifest.md).
+
+_Computation Payload Manifest_ making use of _Outbound Network_ requires either:
+
+1. [Certificate](../../requestor-tutorials/vm-runtime/computation-payload-manifest.md#certificates) trusted by Providers
+2. an instance of a Provider with domain this example will use added to its [domain whitelist](../../provider-tutorials/provider-cli.md#domain-whitelist)
+3. an instance of a Provider with a self signed Certificate imported into its [keystore](../../provider-tutorials/provider-cli.md#keystore)
+
+Following example will show cases 2. and 3. so it will be necesary to start a [local instance of a Provider](../../provider-tutorials/provider-tutorial.md).
```

Comment:
```suggestion
The following example will show cases 2. and 3. so it will be necessary to start a [local instance of a Provider](../../provider-tutorials/provider-tutorial.md).
```